cms/djangoapps/export_course_metadata/toggles.py

```diff
@@ -0,0 +1,21 @@
+"""
+Toggles for export_course_metadata app
+"""
+
+from edx_toggles.toggles import LegacyWaffleFlag, LegacyWaffleFlagNamespace
+
+# Waffle flag to split library to new view.
+# .. toggle_name: export_course_metadata
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Export of course metadata (initially to s3 for use by braze)
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-03-01
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: ??
+# .. toggle_tickets: AA-461
+EXPORT_COURSE_METADATA_FLAG = LegacyWaffleFlag(
```

Comment:
If you're not making an experiment flag or a course waffle but rather just a normal waffle, don't use the legacy waffles. Use the new-style `WaffleFlag` (which doesn't take a separate namespace object, you just say `cms.export_course_metadata`). Look at `edx-toggle` docs.

And since it's not really course-based or user-based, a `WaffleSwitch` might be simpler? But I don't know our rollout plans.

Reply:
thanks! using waffle flag since i'll need to enable for only myself or superusers for testing